compose/queue/api_data/taskqueue_v1beta2.py

```diff
@@ -0,0 +1,626 @@
+        """Create a new Task object.
+
+        Where only the queueName and payloadBase64 are required to
+        make a new object.
+
+        Args:
+            queueName: A string of the name of the associated queue.
+            payloadBase64: A string of the user-defined payload as a
+                string in base64.
+            id: A string of the task id, if None is automatically
+                generated.
+            kind: A string of the kind of object.
+            enqueueTimestamp: A long integer of the timesince the epoch the
+                task was enqueued.
+            leaseTimestamp: A long integer of the time the task becomes
+                avaliable for leasing.
+            retry_count: The number of times a task was leased.
+            tag: The tag associated with the tag.
+        """
+        self.queueName = queueName
+        self.payloadBase64 = payloadBase64
+
+        self.id = id if id is not None else generate_id()
+        self.enqueueTimestamp = enqueueTimestamp if enqueueTimestamp is not None else now()
+        self.leaseTimestamp = leaseTimestamp if leaseTimestamp is not None else now()
+        self.kind = kind
+        self.retry_count = retry_count
+        self.tag = tag
+
+    def _asdict(self):
+        """Convert the current object into a dictionary.
+
+        Returns:
+            A dictionary of properties mapped to keys.
+        """
+        return {
+            "id": self.id,
+            "kind": self.kind,
+            "enqueueTimestamp": self.enqueueTimestamp,
+            "leaseTimestamp": self.leaseTimestamp,
+            "payloadBase64": self.payloadBase64,
+            "queueName": self.queueName,
+            "retry_count": self.retry_count,
+            "tag": self.tag
+        }
+
+    def to_json(self):
+        """Convert the current object into a json dictionary.
+
+        Returns:
+            A dictionary where the keys match the properties of the
+            object.
+        """
+        return self._asdict()
+
+    @staticmethod
+    def from_json(json):
+        """Create a new Task object from a json dictionary.
+
+        Includes error checking that all the required properties
+        are present.
+
+        Args:
+            json: A dictionary where keys map to Task object
+                properties.
+        Returns:
+            The converted Task object.
+        """
+        keys = [
+            "id", "kind", "enqueueTimestamp", "leaseTimestamp", "payloadBase64",
+            "queueName", "retry_count", "tag"
+        ]
+        if any(key not in json for key in keys):
+            raise Exception("Not all values specified.")
+        return Task(**json)
+
+
+class Stats(object):
+    """Describes a Stats object used in the Google TaskQueue.
+
+    The methods for creation from json with error check and automatic
+    property building on creation.
+
+    TODO: We currently do not update our stats objects.
```

Comment:
Only if we ever need to use the stats, they are there for completeness with the API description but are unused as I do not intend to use them.